### PR TITLE
Add "Group Stats" for QM Object Cache Plugin

### PR DESCRIPTION
**Pull request by emrikol**
## Description

These changes introduce a new feature to the Query Monitor plugin, which allows users to view detailed group statistics for the object cache operations. This can help in identifying the performance of different cache groups, and can provide useful insights for performance optimization.  Three files have been added or modified as follows:

* `class-qm-collector-object-cache-group-stats.php`: The `QM_Collector_Object_Cache_Group_Stats` collector class gathers statistics for the group data of object cache operations. It collects data such as operation count, time and size for each group.
* `class-qm-output-html-object-cache-group-stats.php`: The `QM_Output_Html_Object_Cache_Group_Stats` class is responsible for formatting and outputting the data collected by `QM_Collector_Object_Cache_Group_Stats`. It provides a human-readable output for the stats in the footer. It also includes helper methods to convert bytes into human-readable format, convert time to milliseconds, output sortable table columns, and handle table cell outputs.
* `qm-object-cache.php`: This file has been updated to include support for the new "Group Stats" collector and HTML output.

image

### Plugin Updated: qm-plugins

Added a "Group Stats" sub-panel for Object Cache to aid in development and debugging.

## Pre-review checklist

Please make sure the items below have been covered before requesting a review:

- [ ] This change works and has been tested locally (or has an appropriate fallback).
- [x] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change uses a rollout method to ease with deployment (if applicable - especially for large scale actions that require writes).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [x] I've created a changelog description that aligns with the provided examples.

## Pre-deploy checklist

- [ ] VIP staff: Ensure any alerts added/updated conform to internal standards (see internal documentation).

### CODE REVIEW

File: qm-plugins/qm-object-cache/html/class-qm-output-html-object-cache-group-stats.php [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+					$total[ $operation_type ]['count'] += $value['count'];
+					$total[ $operation_type ]['time']  += $value['time'];
+					$total[ $operation_type ]['size']  += $value['size'];
+
+				}
+
+				echo '</tbody>';
+				echo '<tfoot>';
+				echo '<tr>';
```
Comment (GaryJones) [inline qm-plugins/qm-object-cache/html/class-qm-output-html-object-cache-group-stats.php:81]: I think there should be a corresponding `</tr>` somewhere below here.

Comment (emrikol) [reply qm-plugins/qm-object-cache/html/class-qm-output-html-object-cache-group-stats.php:81]: Should be fixed now 🤞

**Review by rinatkhaziev — APPROVED** [review]
Thank you, this is great! 🎉

**Comment by rinatkhaziev** [comment]
I did a round of basic testing, seems to work great

**Pull request merged**